src/app/util/af-event.cpp

```diff
@@ -28,7 +28,6 @@
 
 #include <app-common/zap-generated/callback.h>
 
-#include <zap-generated/af-gen-event.h>
```

Comment:
This makes a bunch of stuff in this file be dead code, no?

If it really is dead code, we should just remove it....